website/docs/patterns/multi-select-patterns/partials/specifications/specifications.md

```diff
@@ -21,19 +21,19 @@ Within the Multi-select pattern itself, use a 16px horizontal gap between the me
 
 ### In context
 
-Within a larger context, use the same spacing recommendation of 16px vertical gap between the pattern, filter elements, and data set.
+Within a larger context, use a 16px vertical gap between the filter elements, multi-selection pattern, and table.
```

Comment:
What's a "larger context"? Do you mean with multiple rows of information in the multi-select pattern?

I would mention vertical spacing vs. horizontal spacing which I think would help clarify what is meant.

Reply:
By larger context I mean when paired with other filtering patterns and functions around the table. I'll do a bit to clarify this